### PR TITLE
Eng 2022 implement operator repo additional tests

### PR DESCRIPTION
**Pull request by eunice-chan**
## Describe your changes and why you are making these changes
Implement the rest of the tests for operator

## Related issue number (if any)
ENG-2022

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [N/A] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [x] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [N/A] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

**Review by saurav-c — COMMENTED** [review]
Can you rebase this?

File: src/golang/lib/repos/tests/require.go [hunk 1]
```diff
@@ -193,3 +194,30 @@ func requireDeepEqualOperators(t *testing.T, expected, actual []models.Operator)
...
+// LoadOperators contain the same elements.
+func requireDeepEqualLoadOperators(t *testing.T, expected, actual []views.LoadOperator) {
+	require.Equal(t, len(expected), len(actual))
+
+	for _, expectedOperator := range expected {
+		found := false
+		var foundOperator views.LoadOperator
+
+		for _, actualOperator := range actual {
+			if (expectedOperator.OperatorName == actualOperator.OperatorName &&
```
Comment (saurav-c) [inline src/golang/lib/repos/tests/require.go:208]: can this explicit checking be replaced with a call to `requireDeepEqual`?

**Review by saurav-c — APPROVED** [review]
LGTM left a comment

File: src/golang/lib/repos/tests/require.go (hunk 1 above)
Comment (eunice-chan) [reply src/golang/lib/repos/tests/require.go:208]: Good suggestion! Updated requireDeepEqualDAGEdges and requireDeepEqualLoadOperators to check with reflect.DeepEqual

**Pull request merged**